app/models/member.rb

```diff
@@ -107,6 +107,15 @@ def has_role?(role_sym)
     roles.any? { |r| r.name.gsub(/\s+/, "_").underscore.to_sym == role_sym }
   end
 
+  def list_roles
```

Comment:
Wait, sorry!  I said I think this is good to merge but I was just looking at the views.

I don't think we need this method here.  Why can't we just use join(", ") here, as we do in many other views?